### PR TITLE
applaunchservices 0.2.0

### PR DESCRIPTION
**Pull request by impact27**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by goanpeca** [comment]
@impact27 there is no LICENSE file
Error:
`License file given in about/license_file (/home/conda/conda-recipes/applaunchservices/LICENSE)`

**Comment by goanpeca** [comment]
Also you could add me to the extra key so I can also help with this recipe in cadse of need ;-)

**Comment by goanpeca** [comment]
@impact27 we should also ask upstream that they package the LICENSE with the .. pakage :-)

**Comment by goanpeca** [comment]
>How do you do that?

Open an issue on where the repository is located and ask if they could include the license file as part of the package in a future release. Also suggest to rename to `LICENSE.txt` to be windows friendly.

File: recipes/applaunchservices/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+
+test:
+  requires:
+    - pytest
+    - pyqt
+  imports:
+    - pytest
+    - applaunchservices
+    - PyQt5
```
Comment (ccordoba12) [inline recipes/applaunchservices/meta.yaml]: The only test necessary here is that `applaunchservices` imports. The other import's are not related to this package.

I understand that these packages are necessary to run the `applaunchservices` test suite, but the tests for conda packages are simpler. It's usually enough to test that the package imports correcly.

File: recipes/applaunchservices/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
```
Comment (ccordoba12) [inline recipes/applaunchservices/meta.yaml]: This section is not necessary.

Comment (impact27) [reply recipes/applaunchservices/meta.yaml]: Are you sure? I think I had a test fail because pytest was not installed.

Comment (ccordoba12) [reply recipes/applaunchservices/meta.yaml]: Yes because this package doesn't depend on pytest directly (only its test suite depends on pytest).

**Comment by goanpeca** [comment]
Even if this package can be built as NoArch, it is only useful on Mac, so should we build it as an arched packaged for OSX only @ccordoba12 ?

**Pull request merged**